src/main/java/com/ericsson/ei/config/SwaggerConfig.java

```diff
@@ -33,25 +34,30 @@
 @EnableSwagger2
 public class SwaggerConfig {
 
+    private static final String CONTACT_NAME = "Eiffel Intelligence Maintainers";
+    private static final String CONTACT_URL = "https://github.com/eiffel-community/eiffel-intelligence";
```

Comment:
https://github.com/orgs/eiffel-community/teams/eiffel-intelligence-maintainers

Reply:
Is it possible to reach this page if you are not a member? If you are not logged in you are redirected to login screen.

Reply:
Yes I also considered that, and left it to be just the GitHub repo. The documentation for Eiffel Intelilgence is located here so I figured we can use this link here, and the link for maintainers can be changed to directly contact @eiffel-community/eiffel-intelligence-maintainers :)

Reply:
The @eiffel-community/eiffel-intelligence-maintainers group is visible to GitHub members as long as they're signed in... Should this link: https://github.com/orgs/eiffel-community/teams/eiffel-intelligence-maintainers  instead be visible from the README in the repo?

Reply:
It is ok for me to have what it is now.